bindinfo/bind_test.go

```diff
@@ -328,39 +328,39 @@ func (s *testSuite) TestGlobalAndSessionBindingBothExist(c *C) {
 	tk.MustExec("create table t2(id int)")
 
 	tk.MustQuery("explain SELECT * from t1,t2 where t1.id = t2.id").Check(testkit.Rows(
-		"HashLeftJoin_8 12487.50 root inner join, inner:TableReader_15, equal:[eq(test.t1.id, test.t2.id)]",
+		"HashLeftJoin_8 12487.50 root inner join, inner:TableReader_15, equal:[eq(Column#1, Column#3)]",
```

Comment:
It is hard to read the result now. Is there no easy way to keep the origin format?

Reply:
We can create the map that recording the column's changing. But it's hard to keep the original result.